src/NoteMouseHandler/SelectionMouseHandler.js

```diff
@@ -106,10 +105,10 @@ const dragSelectionLeftEdgeAction = (ctx) => (onMouseDown, onMouseMove) => {
   const originalFromTick = ctx.selection.fromTick
   let startPosition
 
-  onMouseDown(e => { startPosition = getLocal(e) })
+  onMouseDown(local => { startPosition = local })
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->